config.yml

```diff
@@ -1,25 +1,29 @@
----
-image: jembi/platform
-defaultTargetLauncher: swarm
-disableKubernetes: true
-disableIG: true
-disableCustomTargetSelection: true
+projectName: Cares-on-platform
+image: jembi/platform:latest
```

Comment:
Shouldn't we pin this so that we don't have breaking changes in platform updates affect this package?

Reply:
```suggestion
image: jembi/platform:1.5.0
```